grakn-test/src/test/java/ai/grakn/test/engine/controller/CommitLogControllerTest.java

```diff
@@ -28,30 +28,89 @@
 import ai.grakn.concept.ResourceType;
 import ai.grakn.concept.RoleType;
 import ai.grakn.engine.cache.EngineCacheProvider;
+import ai.grakn.engine.cache.EngineCacheStandAlone;
+import ai.grakn.engine.controller.CommitLogController;
+import ai.grakn.engine.controller.SystemController;
+import ai.grakn.engine.postprocessing.UpdatingInstanceCountTask;
+import ai.grakn.engine.tasks.TaskManager;
+import ai.grakn.engine.tasks.manager.StandaloneTaskManager;
+import ai.grakn.engine.util.EngineID;
 import ai.grakn.exception.GraknValidationException;
 import ai.grakn.factory.SystemKeyspace;
 import ai.grakn.graph.admin.ConceptCache;
-import ai.grakn.test.EngineContext;
 import ai.grakn.util.REST;
 import ai.grakn.util.Schema;
+import com.jayway.restassured.RestAssured;
 import com.jayway.restassured.http.ContentType;
 import org.junit.After;
+import org.junit.AfterClass;
 import org.junit.Before;
-import org.junit.ClassRule;
+import org.junit.BeforeClass;
 import org.junit.Test;
 
 import java.util.UUID;
+import spark.Service;
 
+import static ai.grakn.engine.GraknEngineServer.configureSpark;
+import static ai.grakn.test.GraknTestEnv.ensureCassandraRunning;
+import static ai.grakn.util.REST.Request.COMMIT_LOG_COUNTING;
+import static ai.grakn.util.REST.Request.KEYSPACE;
+import static com.jayway.restassured.RestAssured.baseURI;
 import static com.jayway.restassured.RestAssured.delete;
 import static com.jayway.restassured.RestAssured.given;
 import static org.junit.Assert.assertEquals;
+import static org.mockito.ArgumentMatchers.argThat;
+import static org.mockito.Mockito.atLeastOnce;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.spy;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
 
 public class CommitLogControllerTest {
-    private final String KEYSPACE = "test";
     private final ConceptCache cache = EngineCacheProvider.getCache();
 
-    @ClassRule
-    public static final EngineContext engine = EngineContext.startInMemoryServer();
+    private static final String TEST_KEYSPACE = "test";
+    private static final int PORT = 4567;
+
+    private static Service spark;
+    private static TaskManager manager;
+
+    @BeforeClass
+    public static void setupControllers() throws Exception {
+        ensureCassandraRunning();
+
+        EngineCacheProvider.init(EngineCacheStandAlone.getCache());
+
+        baseURI = "http://localhost:4567";
```

Comment:
Nitpick police. you should use the variable `PORT` to prevent hardcoding more stuff